packages/cardano-services/src/Projection/prepareTypeormProjection.ts

```diff
@@ -39,13 +41,14 @@ import { passthrough } from '@cardano-sdk/util-rxjs';
  *
  */
 export enum ProjectionName {
+  Address = 'address',
   Asset = 'asset',
   Handle = 'handle',
   StakePool = 'stake-pool',
   StakePoolMetadataJob = 'stake-pool-metadata-job',
   StakePoolMetricsJob = 'stake-pool-metrics-job',
-  UTXO = 'utxo',
-  Address = 'address'
+  StakePoolRewardsJob = 'stake-pool-rewards-job',
```

Comment:
We should probably add a migration adding the required table in this commit

Reply:
5303986e3 should solve this